packages/opentelemetry-tracing/src/Span.ts

```diff
@@ -86,8 +88,22 @@ export class Span implements api.Span, ReadableSpan {
     return this.spanContext;
   }
 
+  setAttribute(key: string, value: AttributeValue): this;
   setAttribute(key: string, value: unknown): this {
     if (this._isSpanEnded()) return this;
+    if (key.length === 0) {
+      this._logger.warn(`Invalid attribute key: ${key} value=${value}`);
```

Comment:
nit: stick to `:` or `=`

Reply:
oops I meant to remove value entirely as it is a potential security concern to log values